zipkin-scrooge/src/main/scala/com/twitter/zipkin/conversions/thrift.scala

```diff
@@ -130,8 +130,8 @@ object thrift {
     lazy val toThrift = thriftscala.Dependencies(d.startTs, d.endTs, d.links.map(_.toThrift))
   }
   class ThriftDependencies(d: thriftscala.Dependencies) {
-    lazy val toDependencies = Dependencies(d.startTs, d.endTs, d.links.map(_.toDependencyLink))
+    lazy val toDependencies: Dependencies = Dependencies(d.startTs, d.endTs, d.links.map(_.toDependencyLink))
```

Comment:
something magical happened where after I added the scrooge dep to the query project, this started to fail compile unless I added a type hint